### PR TITLE
Load / show fees for cross-chain deposits

### PR DESCRIPTION
**Pull request by veado**
- [x] Add const `BASE_CHAIN` + `BASE_CHAIN_ASSET` - base chain == chain RUNE is running on (currently BNC)
- [x] Load / show `pool` and/or `base` fees (base == fees for base-chain /  pool == fees for cross-chain)
- [x] `emptyFunc` for lazy devs
- [x] Fix `sequenceT...` helpers (review suggestion)

Close ##552

### CODE REVIEW

File: src/renderer/services/chain/fees.ts [hunk 1]
```diff
@@ -34,7 +36,21 @@ const { stream$: reloadFees$, trigger: reloadFees } = triggerStream()
 
 // reload fees
 Rx.combineLatest([selectedPoolChain$, reloadFees$])
-  .pipe(RxOp.switchMap(([oChain]) => Rx.of(FP.pipe(oChain, O.map(reloadFeesByChain)))))
+  .pipe(
+    RxOp.switchMap(([oChain]) =>
+      Rx.of(
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/services/chain/fees.ts:41]: do we really need new stream here ?
seems simple `RxOp.tap` is enough ?

File: src/renderer/components/stake/add/AddStake.tsx [hunk 2]
```diff
@@ -198,13 +216,16 @@ export const AddStake: React.FC<Props> = ({
...
             <Col>
-              <Styled.ReloadFeeButton onClick={reloadFee} disabled={RD.isPending(feeRD)}>
+              <Styled.ReloadFeeButton onClick={reloadFees} disabled={RD.isPending(fees)}>
                 <SyncOutlined />
               </Styled.ReloadFeeButton>
             </Col>
             <Col>
-              <Styled.FeeLabel disabled={RD.isPending(feeRD)}>
-                {intl.formatMessage({ id: 'common.fee' })}: {feeLabel}
+              <Styled.FeeLabel disabled={RD.isPending(fees)}>
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/stake/add/AddStake.tsx:224]: maybe we could use `loading` prop here ?

File: src/renderer/components/stake/add/AddStake.tsx [hunk 3]
```diff
@@ -147,36 +146,55 @@ export const AddStake: React.FC<Props> = ({
...
-    (fee: BaseAmount) =>
+    (fee: BaseAmount, asset) =>
       formatAssetAmountCurrency({
         amount: baseToAsset(fee),
-        asset: O.toUndefined(oChainAsset),
+        asset,
         trimZeros: true
       }),
-    [oChainAsset]
+    []
```
Comment (thatStrangeGuyThorchain) [inline src/renderer/components/stake/add/AddStake.tsx:156]: as it does not depend on anything we could move it to the helper file

**Review by thatStrangeGuyThorchain — APPROVED** [review]

File: src/renderer/components/stake/add/AddStake.tsx (hunk 3 above)
Comment (veado) [reply src/renderer/components/stake/add/AddStake.tsx:156]: ok

File: src/renderer/components/stake/add/AddStake.tsx (hunk 2 above)
Comment (veado) [reply src/renderer/components/stake/add/AddStake.tsx:224]: We would lost `Fee:` text before fee amounts then.

**Pull request merged**